api/src/main/java/org/pmiops/workbench/api/CohortReviewController.java

```diff
@@ -36,15 +54,113 @@
                             .status(participant.getStatus());
                 }
             };
 
+    @Autowired
+    CohortReviewController(CohortReviewDao cohortReviewDao,
+                           CohortDao cohortDao,
+                           ParticipantCohortStatusDao participantCohortStatusDao,
+                           BigQueryService bigQueryService,
+                           ParticipantCounter participantCounter) {
+        this.cohortReviewDao = cohortReviewDao;
+        this.cohortDao = cohortDao;
+        this.participantCohortStatusDao = participantCohortStatusDao;
+        this.bigQueryService = bigQueryService;
+        this.participantCounter = participantCounter;
+    }
+
     /**
-     * Get all participants for the specified cohortId and cdrVersionId.
+     * Get the cohort review info. If a cohort review does not exist, a default review will be created without
+     * any participants. The {@link CohortReviewInfo#getReviewCount()} will equal zero (0) and be the indicatior
+     * that the cohort review has not been fully created. If a cohort review does exist, the
+     * {@link CohortReviewInfo#getReviewCount()} will be > 0. If a cohort definition for the specified cohortId
+     * doesn't exist a {@link BadRequestException} will be thrown.
      *
      * @param cohortId
      * @param cdrVersionId
+     * @return
+     */
+    @Override
+    public ResponseEntity<CohortReviewInfo> getCohortReviewInfo(Long cohortId, Long cdrVersionId) {
+
+        CohortReview cohortReview = cohortReviewDao.findCohortReviewByCohortIdAndCdrVersionId(cohortId, cdrVersionId);
+        if (cohortReview == null) {
+            CohortDefinition definition = cohortDao.findCohortByCohortId(cohortId);
+            if (definition == null) {
+                throw new BadRequestException("Invalid Request: No Cohort definition matching cohortId: " + cohortId);
+            }
+            SearchRequest request = new Gson().fromJson(definition.getCriteria(), SearchRequest.class);
+            QueryResult result = bigQueryService.executeQuery(bigQueryService.filterBigQueryConfig(participantCounter.buildParticipantCounterQuery(request)));
+            Map<String, Integer> rm = bigQueryService.getResultMapper(result);
+            List<FieldValue> row = result.iterateAll().iterator().next();
+            long cohortCount = bigQueryService.getLong(row, rm.get("count"));
+
+            cohortReview = new CohortReview();
+            cohortReview.setCohortId(cohortId);
+            cohortReview.setCdrVersionId(cdrVersionId);
+            cohortReview.matchedParticipantCount(cohortCount);
+            cohortReview.setCreationTime(new Timestamp(System.currentTimeMillis()));
+            cohortReviewDao.save(cohortReview);
+        }
+
+        long count = participantCohortStatusDao.countByParticipantKey_CohortReviewId(cohortReview.getCohortReviewId());
+        CohortReviewInfo info = new CohortReviewInfo()
```

Comment:
Why not return the whole CohortReview object (with reviewed count) here?

Seems like the other metadata might be something we'd want to display...